test/e2e/atlas/kubernetes_config_generate_test.go

```diff
@@ -131,6 +131,28 @@ func InitialSetup(t *testing.T) KubernetesConfigGenerateProjectSuite {
 	return s
 }
 
+func TestProjectWithWrongOrgID(t *testing.T) {
+	s := InitialSetup(t)
+	cliPath := s.cliPath
+
+	t.Run("should fail to generate when the org id does not match the owner of the project", func(t *testing.T) {
```

Comment:
this feels like it could be a unit and not an e2e whcih should be faster to run

Reply:
Done